pyproject.toml

```
@@ -61,7 +61,7 @@ dependencies = [
   "shapely >=2.0",
   "sparse",
   "toolz",
-  "xarray >=2023.11.0, !=2024.6.0, <=2025.3.0", # FIXME: Remove the highest version when rechunker is updated.
+  "xarray >=2023.11.0, !=2024.6.0, <=2025.4.0", # FIXME: Remove the highest version when rechunker is updated.
```

```suggestion
  "xarray >=2023.11.0, !=2024.6.0
```
Rechunker was updated!